From 5c17c0d0d1f86cb65f1f95f4e20abfc2f4f2741c Mon Sep 17 00:00:00 2001
From: Reagan Bohan <xbjfk.github@gmail.com>
Date: Mon, 5 May 2025 07:55:39 +0000
Subject: [PATCH 38/80] ci: add quotes to URL in install-musl.sh script

This silences shellcheck warnings

(backport <https://github.com/rust-lang/libc/pull/4443>)
(cherry picked from commit 60f7b3d1841ca463cdfb1af85b713efd0b4d3b29)
---
 ci/install-musl.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ci/install-musl.sh b/ci/install-musl.sh
index 8567c0848..d3752a900 100755
--- a/ci/install-musl.sh
+++ b/ci/install-musl.sh
@@ -17,7 +17,7 @@ esac
 musl="musl-${musl_version}"
 
 # Download, configure, build, and install musl:
-curl --retry 5 https://www.musl-libc.org/releases/${musl}.tar.gz | tar xzf -
+curl --retry 5 "https://www.musl-libc.org/releases/${musl}.tar.gz" | tar xzf -
 
 cd "$musl"
 case ${1} in
-- 
2.43.0

